examples/update_catalog.java

```diff
@@ -0,0 +1,121 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
```

Comment:
lets move this to jbang-examples repo

possibly make it have a bit more structure.

Reply:
M;ore structure? You mean the code or the repo?

Reply:
meant the repo; been pondering if we by simple means could make a simple example repo that is bette than just one long java file list.

Reply:
Sure, we could def do that. The catalog file can support any hierarchy (as seen from the point where the catalog json file lives)